python/pyarrow/tests/parquet/test_pandas.py

```diff
@@ -0,0 +1,759 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
these aren't pandas-dependent tests - rather they're the tests that test interop with pandas data structures